src/shogun/mathematics/Math.h

```diff
@@ -658,7 +658,8 @@ class CMath : public CSGObject
 		COMPLEX128_ERROR_ONEARG(atan)
 
 		/** Computes arc tangent with 2 parameters
-		 * @param x input
+		 * @param x input(numerator)
+		 * @param y input(denominator)
 		 * @return arctan(x/y)
 		 */
 		static inline float64_t atan2(float64_t x, float64_t y)
```

Comment:
If we are doing changes here, I think it might be worth to change this to atan2(y,x) since it is much more common imo.